source/components/organisms/SubstepList/SubstepList.tsx

```diff
@@ -0,0 +1,255 @@
+                if (value[item.key] && value[item.key][shownField.fieldId]) {
+                  const term = parseFloat(value[item.key][shownField.fieldId]);
+                  // eslint-disable-next-line no-restricted-globals
+                  if (isNaN(term)) {
+                    return partialSum;
+                  }
+                  return partialSum + term;
+                }
+                return partialSum;
+              }, 0)
+            : sum,
+        0
+      );
+      listItems.push({
+        category: 'sum',
+        component: <LargeText>{`${sum} kr`}</LargeText>,
+      });
+    }
+  }
+
+  return (
+    <Wrapper>
+      {listItems.length > 0 && (
+        <GroupedList
+          heading={heading}
+          items={listItems}
+          categories={categories}
+          color={color}
+          onEdit={() => setEditable(!editable)}
+        />
+      )}
+      {editable && !summary && (
+        <>
+          <FieldLabelContainer>
+            <FieldLabel color={color} help={buttonHelp}>
+              LÄGG TILL
+            </FieldLabel>
+          </FieldLabelContainer>
+          <ScrollView horizontal>
+            {items.map((item: Record<string, any>, index: number) => (
+              <SubstepButton
+                key={`${index}-${item.title}`}
+                text={item.title}
+                iconName="add"
+                iconColor={colors.substepList[color].addButtonIconColor}
+                value={value[item.key] || {}}
+                color={colors.substepList[color].addButtonColor}
+                onChange={updateAnswer(item.key)}
+                formId={item.formId}
+                size="medium"
+              />
+            ))}
+          </ScrollView>
+        </>
+      )}
+    </Wrapper>
+  );
+};
+
+SubstepList.propTypes = {
+  /**
+   * The header text of the list.
+   */
+  heading: PropTypes.string,
+  /**
+   * List of all items, corresponding to all subforms
+   */
+  items: PropTypes.array,
+  /**
+   * The categories of the grouping
+   */
+  categories: PropTypes.array,
+  /**
+   * The values of the entire list object
+   */
+  value: PropTypes.object,
+  /**
+   * What should happen to update the values
+   */
+  onChange: PropTypes.func,
+  /**
+   * If the list acts as a summary; default is false.
+   */
+  summary: PropTypes.bool,
+  /**
+   * Sets the color scheme of the list. default is red.
+   */
+  color: PropTypes.string,
+  /** help-object for the questionmark on the label above the buttons to add things. */
+  buttonHelpUrl: PropTypes.string,
+  /** help-object for the questionmark on the label above the buttons to add things. */
+  buttonHelpText: PropTypes.string,
+  /** help-object for the questionmark on the label above the buttons to add things. */
+  buttonHelpHeading: PropTypes.string,
+  /** help-object for the questionmark on the label above the buttons to add things. */
+  buttonHelpTagline: PropTypes.string,
+};
```

Comment:
Don't know if this would break anything, but since we are defining the Props interface this feels lika redundant code. So it would be preferable to remove this and just keep the defaultProps :D